File: Prática/Lista 01 - Java /Ex04/src/ex04/Ex04.java
package ex04;

import java.util.Scanner;

/**
 * @author bryan
 */

public class Ex04 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        boolean loop = true;

        do {

            System.out.println("1. Soma de dois números");
            System.out.println("2. Raiz quadrada de um número");
            System.out.println("0. Sair");

            System.out.print("Digite uma opção: ");
            byte opc = sc.nextByte();

            switch (opc) {

                case 1:

                    System.out.print("Digite dois números para soma-los: ");

                    double a = sc.nextDouble();
                    double b = sc.nextDouble();
                    
                    sc.close();

                    System.out.printf("%.0f + %.0f = %.0f \n", a, b, (a + b));

                    break;

                case 2:

                    System.out.print("Digite um número para obter sua raiz: ");

                    int c = sc.nextInt();
                    
                    sc.close();

                    System.out.printf("A raiz de %d é %.0f \n", c, Math.sqrt(c));

                    break;

                case 0:

                    loop = false;

                    break;

                default:

                    System.out.println("Opção inválida!");

                    break;
            }

        } while (loop == true);
    }

}

File: Prática/Lista 04 - C/Ex04/Ex04.c
#include "../Console.c"
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

float soma(float a, float b)
{
    return a + b;
}

void menu()
{
    printf("1. Soma de dois números\n");
    printf("2. Raiz quadrada de um número\n");
    printf("\n");
}

int main()
{
    int opcao;
    float numA, numB, resultado;

    clear();
    menu();

    printf("Digite a opção desejada: ");
    scanf("%d", &opcao);
    clear();

    switch (opcao)
    {
    case 1:
        printf("Digite o primeiro número: ");
        scanf("%f", &numA);
        printf("Digite o segundo número: ");
        scanf("%f", &numB);
        resultado = soma(numA, numB);
        clear();
        printf("%g + %g = %g\n\n", numA, numB, resultado);
        break;

    case 2:
        printf("Digite um número para calcular a raiz: ");
        scanf("%f", &numA);
        resultado = sqrt(numA);
        clear();
        printf("A raiz de %g é %g\n\n", numA, resultado);
        break;

    default:
        printf("Opção inválida\n\n");
        break;
    }

    return 0;
}
